src/processors/JsonSchemaInputProcessor.ts

```diff
@@ -15,9 +25,17 @@ export class JsonSchemaInputProcessor extends AbstractInputProcessor {
      * 
      * @param schema to simplify to common model
      */
-    static convertSchemaToCommonModel(schema: any) : Map<string, CommonModel>{
-        //To be defined in https://github.com/asyncapi/shape-up-process/issues/53
-        throw new Error('Method not implemented.');
+    static convertSchemaToCommonModel(schema: Schema | boolean) : {[key: string]: CommonModel} {
+        const commonModels = simplify(schema);
+        const commonModelsMap : {[key: string]: CommonModel}  = {};
+        commonModels.forEach((value) => {
+            if(value.$id){
+                commonModelsMap[value.$id] = value;
+            }else{
+                console.log("Looks like someone is missing an id");
```

Comment:
Haha 😄

Reply:
yea, placeholder for now until we handle errors correctly. We also need to resolve refs and assign anonymous schemas like our parser. But thats later 😄